numpy/core/getlimits.py

```diff
@@ -337,8 +337,9 @@ class finfo:
         The approximate decimal resolution of this type, i.e.,
         ``10**-precision``.
     tiny : float
-        The smallest positive usable number.  Type of `tiny` is an
-        appropriate floating point type.
+        The smallest positive floating point number with there being no
+        leading 0's in the mantissa. Type of `tiny` is an appropriate 
+        floating point type.
```

Comment:
```suggestion
        The smallest positive non-subnormal (see Notes) floating point
        number. Type of `tiny` is an appropriate floating point type.
```
This is just my opinion, but I have a slight preference for leaving the details of denormal numbers (i.e. "being no leading 0's in the mantissa") out of the parameter description and instead refer to greater levels of detail outside of the parameters section.

LMK what you think!

Reply:
@rossbar That is fine with me, too.

Please take note that I used the phrase "being no leading 0's in the mantissa" since it already appeared in the description of another parameter in the same docstring. Cf. `minexp`.

To be keep it consistent, I guess, we should modify the phrase there as well. What do you think?

Reply:
Yeah, I was suprised how often we used "mantissa" since I am not sure its a good start for a general audience. I liked the "with full precision", but overall, I guess a see notes is just the right way to explain it.

Mainly commenting, because it seemed to me like we can just drop the current second sentence, saying that it gives out the correct type seems pretty irrelevant (all of these are obviously returned as the same type as is being queried).

Reply:
As for "mantissa" - I think we should prefer ["significand"](https://en.wikipedia.org/wiki/Significand). See also this [stronger opinion](https://en.wikipedia.org/wiki/Talk:Significand) against mantissa

Reply:
Okay, I tried to combine these comments in the most recent commit. In particular, I tried
to avoid usage of the word mantissa if possible.

Regarding the matter "significand" vs. "mantissa": I have no strong feelings about this. The former is used in the IEEE 754-standard, while the latter seems to be a bit more common in the scientific computing literature. But I have no overview how often and where mantissa is in use within the NumPy documentation. For instance, `finfo` has the attribute `nmant` which would need to be renamed, too. 

So, this topic might be a little beyond the scope of this PR.